src/util/QtSingleApplication.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
```

Comment:
Are we sure that a server-socket based approach is what we are looking for single instance application? Are there no easier alternatives? 
(and am not talking about .lock files, those are messy; keeps creating problems)

Reply:
> @LordAmit Any other thoughts or directions we want to take on this? I can change the UUID to just be `Brightness Controller`. I am not sure there is a better way in Qt4 for a solid single instance application

Hmm, we can use this for now and look for better approaches; otherwise, things won't be moving. 

Sorry for the wait! :dancer: